recipes/node2vec/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "node2vec" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "28c4e21841db1224b9d7818c177c721bc5244b0004f57e143003ba78243dab04" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - networkx==1.11
+    - numpy==1.11.2
+    - gensim==0.13.3
+  run:
+    - python
+    - networkx==1.11
```

Comment:
I think the dependency list should actually come from [here](https://github.com/eliorc/node2vec/blob/2ff4bf481bae0a2120d66c350254dbd74137ab9e/setup.py#L12-L18).